### DISCUSSION TITLE
New donation method

### DISCUSSION THREAD

**Discussion by nh2** [bromite/bromite#2420] [General]
Hi, the ability to donate via PayPal is no longer available.

Could you set up another non-cryptocurrency donation method, such as OpenCollective or Patreon?

Thanks!

**Comment by csagan5** [comment]
I will finish setting up Patreon in the next days; PayPal is no longer an option as they requested documentation of my business and services/products, but neither exist so I cannot possibly comply.

**Reply by csagan5** [reply]
I have finished setting up Patreon: https://www.patreon.com/csagan5

**Reply by nh2** [reply]
Great, I just switched my backing to it.

Once it is [available](https://support.patreon.com/hc/en-us/articles/360041721372-Annual-memberships-creator-overview) for you in the future, also consider offering yearly support plans, as that can reduce the accounting overhead for small companies that want to back you.

**Reply by csagan5** [reply]
> Great, I just switched my backing to it.

Thanks!

> Once it is [available](https://support.patreon.com/hc/en-us/articles/360041721372-Annual-memberships-creator-overview) for you in the future, also consider offering yearly support plans, as that can reduce the accounting overhead for small companies that want to back you.

I am new to the current Patreon (it has changed a lot since last time I used it); it is already available but it says that once switching it on I cannot turn it off. I am evaluating whether it is a problem or not for people that want to pledge temporarily for a month.
